queenbee/io/common.py

```diff
@@ -14,7 +14,7 @@
 
 class ItemType(str, Enum):
```

Comment:
Should we make enum items as same as the what we have for DAG inputs:
For example, here is missing: File, Path,
Renaming: Any =>Generic, Object => JSONObject

Reply:
That's a good idea. Will do that.

Reply:
https://github.com/ladybug-tools/queenbee/pull/170/commits/a1af031c1ef4bf6abf6e05eb70dcedde294389a8

Reply:
Did you forget "File", "Path"?